sensu/plugins/check-network-traffic.rb

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env /opt/sensu/embedded/bin/ruby
```

Comment:
We shouldn't supply a full path to `/usr/bin/env`. This should be either `#!/usr/bin/env ruby` or `#!/opt/sensu/embedded/bin/ruby`